pkg/client_data/lib/account_api.dart

```diff
@@ -2,11 +2,51 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:convert';
+
 import 'package:json_annotation/json_annotation.dart';
 import 'package:meta/meta.dart';
 
 part 'account_api.g.dart';
 
+/// Session data to be cached from the authenticated client.
+@JsonSerializable()
+class ClientSessionData {
+  final String imageUrl;
+
+  ClientSessionData({
+    @required this.imageUrl,
+  });
+
+  factory ClientSessionData.fromJson(Map<String, dynamic> json) =>
+      _$ClientSessionDataFromJson(json);
+
+  Map<String, dynamic> toJson() => _$ClientSessionDataToJson(this);
+}
+
+/// The server-provided status of the current session.
+@JsonSerializable()
+class ClientSessionStatus {
+  final bool changed;
```

Comment:
A small doc-comment please... `/// True, if the user session has been updated and the current page should be reloaded`